security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -58,6 +59,8 @@ var _ model.XdsResourceGenerator = &sdsservice{}
 
 func NewXdsServer(stop chan struct{}, gen model.XdsResourceGenerator) *xds.DiscoveryServer {
 	s := xds.NewXDS(stop)
+	// No ratelimit for SDS calls in agent.
+	s.DiscoveryServer.RequestRateLimit = rate.NewLimiter(rate.Inf, 1)
```

Comment:
how about not setting limit at all

Reply:
That would be good. I see we already handle zero limit https://github.com/istio/istio/blob/247e396dd30635ba44e81f324f4224967b75e02a/pilot/pkg/xds/discovery.go#L662